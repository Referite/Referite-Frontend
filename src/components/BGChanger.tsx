import { useEffect } from 'react';

function BGChanger() {
    useEffect(() => {
        document.body.style.minHeight = '100vh'
        document.body.style.margin = '0';
        document.body.style.width = '100%';
        document.body.style.height = '100%';

        document.body.style.backgroundColor = '#f3f3f3';
    
        return () => {
            document.body.style.backgroundColor = '';
        };
      }, []);

    return (
        <> </> 
    )
}

export default BGChanger