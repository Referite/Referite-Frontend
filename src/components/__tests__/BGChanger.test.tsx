import renderer from 'react-test-renderer';
import BGChanger from '../BGChanger';

jest.mock('../BGChanger', () => 'BGChanger');
describe('BGChanger Component', () => {
    it('renders correctly', () => {
      const component = renderer.create(
        <BGChanger />
      );
  
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });