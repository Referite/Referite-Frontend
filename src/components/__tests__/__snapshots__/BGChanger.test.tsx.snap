// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BGChanger Component renders correctly 1`] = `<BGChanger />`;
